scripts/ci-conformance.sh

```diff
@@ -93,4 +94,8 @@ cleanup() {
 
 trap cleanup EXIT
 
-make test-conformance
+if [[ "${WINDOWS}" == "true" ]]; then
+  make test-windows-upstream
```

Comment:
nit: what do you think about `make test-conformance-windows`?

Reply:
I named it this way since there is no official "Conformance" for windows.  We run a good set of conformance test but not all and additional Windows tests.

https://github.com/kubernetes/kubernetes/issues/96639 for more info on where we are with Conformance for Windows